### ISSUE TITLE
BASEURL_PLACEHOLDER present in output image links

### ISSUE THREAD

**Issue by noisecapella** [mitodl/ocw-to-hugo#377]
### Steps to Reproduce

Go to https://ocwnext.odl.mit.edu/courses/18-06sc-linear-algebra-fall-2011/pages/positive-definite-matrices-and-applications/similar-matrices-and-jordan-form/ and look at the text "Figure excerpted from Introduction to Linear Algebra". That is the alt text for an image. However the image URL is `BASEURL_PLACEHOLDER/resources/3_4`. `BASEURL_PLACEHOLDER` is an internal sentinel value which should be replaced with `{{ baseurl }}` when it is written out

I think we could fix this in the turndown callbacks by adding something that checks for `img` elements. However, maybe there's a way we can fix this and also fix #352 by replacing the URL with its S3 link

**Comment by gumaerc** [comment]
First of all, the image here is in a table with a single row that isn't being transformed into a markdown table properly, which explains the visible pipes on the page:

input:
```
    <table class="sc_overview">
      <tbody>
        <tr>
          <td><img width="100" height="100" alt="Figure excerpted from 'Introduction to Linear Algebra' by G.S.
              Strang" src="./resolveuid/bfd7faabe01671d68d35acd3909a0756" /></td>
          <td>
            <p>After a final discussion of positive definite matrices, we learn about &quot;similar&quot; matrices:
              <em>B</em> = <em>M<sup>&minus;1</sup>AM</em> for some invertible matrix <em>M</em>. Square matrices can be
              grouped by similarity, and each group has a &quot;nicest&quot; representative in <em>Jordan normal
                form</em>. This form tells at a glance the eigenvalues and the number of eigenvectors.</p>
          </td>
        </tr>
      </tbody>
    </table>
```

output:
```
| ![Figure excerpted from 'Introduction to Linear Algebra' by G.S. Strang](BASEURL_PLACEHOLDER/resources/3_4) |  {{< br >}}{{< br >}} After a final discussion of positive definite matrices, we learn about "similar" matrices: _B_ = _M−1AM_ for some invertible matrix _M_. Square matrices can be grouped by similarity, and each group has a "nicest" representative in _Jordan normal form_. This form tells at a glance the eigenvalues and the number of eigenvectors. {{< br >}}{{< br >}}  
```

The actual issue described here though, the `BASEURL_SHORTCODE` sentinel value being passed through, has to do with the turndown rule only applying to links and not the source of images.  That I believe can be fixed with a simple PR, but with the table issue we might want to consider a PR to change the `ocw-to-hugo` output to use the table shortcode system we created for use with `ocw-studio`.
